docs/recipes/browser-testing.md

```diff
@@ -7,29 +7,33 @@ An example of this is React, at least if you want to use ReactDOM.render and sim
 
 This recipe works for any library that needs a mocked browser environment.
 
-## Install jsdom
+## Install node-browser-environment
 
-Install [jsdom](https://github.com/tmpvar/jsdom).
+Install [node-browser-environment](https://github.com/lukechilds/node-browser-environment).
 
-> A JavaScript implementation of the WHATWG DOM and HTML standards, for use with node.js
+> Simulates a global browser environment using jsdom.
 
 ```
-$ npm install --save-dev jsdom
+$ npm install --save-dev node-browser-environment
```

Comment:
Good spot on the deprecation, yeah I'll rename it. If you wanna hold off on merging this for now I'll update you when it's sorted 👍

Quick Q: Why bump major first? Wouldn't it be better to notify existing users? If they've installed with the default `^` range they'll be expecting to get SemVer minor/patch updates automatically.